src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseEntitySelectPage.tsx

```diff
@@ -18,6 +18,22 @@ import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
 import type {QBOReimbursableExportAccountType} from '@src/types/onyx/Policy';
 
+function Footer({isTaxEnabled, isLocationsEnabled}: {isTaxEnabled: boolean; isLocationsEnabled: boolean}) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    if (!isTaxEnabled && !isLocationsEnabled) {
+        return null;
+    }
+
+    return (
+        <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter, styles.gap2, styles.mt1]}>
+            {isTaxEnabled && <Text style={[styles.mutedNormalTextLabel, styles.pt2]}>{translate('workspace.qbo.outOfPocketTaxEnabledDescription')}</Text>}
+            {isLocationsEnabled && <Text style={styles.mutedTextLabel}>{translate('workspace.qbo.locationsAdditionalDescription')}</Text>}
```

Comment:
For cases involving `Locations[Imported]` and when the current selection is `Journal Entry`, we hide the options for `Check` and `Vendor Bill` which is correct. 
However, I think we need to have similar wording like `outOfPocketTaxEnabledDescription` but we have used `locationsAdditionalDescription` here which seems incorrect. 

![Screenshot 2024-05-27 at 8 20 26 AM](https://github.com/Expensify/App/assets/3004659/74ca3966-fba7-4b26-9d71-8d7a34ea2bc7)

Reply:
you're right!

<img width="400" alt="Screenshot 2024-05-27 at 10 59 42 AM" src="https://github.com/Expensify/App/assets/98560306/24ef035d-2480-4642-bae6-f8d1d5987767">

Reply:
somehow, the correct text that supposed to be here was added in the Account Select page in [this PR](https://github.com/Expensify/App/commit/3247123e08c67303902351d7c8b1b4482f6a15e1) 🤔

Reply:
@hayata-suenaga Oh! And I just noticed that the statement `As you import Locations from, this this export option is unavailable.` is not grammatically correct. It is better to fix this here.

Reply:
I'll asking for the correct text. we don't have to block the PR on this though.

nice catch by the way 😄

Reply:
<img width="400" alt="Screenshot 2024-05-29 at 3 23 27 PM" src="https://github.com/Expensify/App/assets/98560306/652c3846-9433-448c-ab1d-69d0a7ba697a">

Reply:
<img width="400" alt="Screenshot 2024-05-29 at 3 25 14 PM" src="https://github.com/Expensify/App/assets/98560306/99b772f9-e313-4d4c-a16f-2b111f0bec33">

Reply:
updated the UI text